src/fides/api/ops/service/privacy_request/request_runner_service.py

```diff
@@ -516,3 +531,74 @@ def generate_id_verification_code() -> str:
     Generate one-time identity verification code
     """
     return str(random.choice(range(100000, 999999)))
+
+
+def _retrieve_child_results(  # pylint: disable=R0911
+    fides_connector: Tuple[str, ConnectionConfig], rule_key: str
+) -> Optional[Dict[str, Optional[List[Row]]]]:
+    """Get child access request results to add to upload."""
+    secrets = fides_connector[1].secrets
```

Comment:
not a huge deal, but on first glance i'm thinking you may want to just instantiate a real `FidesConnector` and use some of its methods to initialize and authenticate a `FidesClient`? just may help to DRY this up a bit over the codebase so you don't have to go through the secrets parsing and login/client instantiation again